specification/common-types/resource-management/v6/privatelinks.json

```diff
@@ -42,11 +43,11 @@
           "description": "Array of private endpoint connections.",
           "items": {
             "$ref": "#/definitions/PrivateEndpointConnection"
-          }
+          },
+          "x-ms-identifiers": []
```

Comment:
I understand this probably makes sense for backcompat reasons.

Technically... I think they do have some unique identifier properties actually. But it'd be something like `.properties.privateEndpoint.id`?

Reply:
it is not part of properties it is extending directly from resources so I changed this to: `"x-ms-identifiers": ["id"]`